app/helpers/application_helper.rb

```diff
@@ -7,14 +7,21 @@ def content_tagger_url
     Plek.new.external_url_for("content-tagger")
   end
 
-  def preview_url(slug, auth_bypass_id: nil)
-    url = "#{Plek.new.external_url_for('draft-origin')}/#{slug.sub(/^\//, '')}"
-
-    if auth_bypass_id.present?
-      url = JwtHelper.access_limited_preview_url(url, auth_bypass_id)
-    end
+  def authenticated_preview_url(slug)
+    "#{Plek.new.external_url_for('draft-origin')}/#{slug.sub(/^\//, '')}"
+  end
 
-    url
+  def anonymous_preview_url(step_by_step_page, user)
```

Comment:
As this is now coupled to a step by step should it have step_by_step in the function name?